plugins/woocommerce-blocks/packages/checkout/slot/index.js

```diff
@@ -1,41 +1,22 @@
 /**
  * External dependencies
  */
-import deprecated from '@wordpress/deprecated';
 import { CURRENT_USER_IS_ADMIN } from '@woocommerce/settings';
 import { Children, cloneElement } from '@wordpress/element';
+// It is very important to export this directly from the build module to avoid introducing side-effects
+// from importing the index of the @wordpress/components package.
+// eslint-disable-next-line -- When adding comments to imports it breaks the external/internal dependencies lint.
 import {
 	createSlotFill as baseCreateSlotFill,
-	__experimentalUseSlot,
-	useSlot as __useSlot, //eslint-disable-line
-} from 'wordpress-components';
+	useSlot,
+	useSlotFills,
+} from 'wordpress-components-slotfill/build-module/slot-fill';
 
 /**
  * Internal dependencies
  */
 import BlockErrorBoundary from '../components/error-boundary';
 
-/**
- * This function is used in case __experimentalUseSlot is removed and useSlot is not released, it tries to mock
```

Comment:
This stuff doesn't make such sense when we peg the version of slot fill that we import. maybe we pegged the version after we added this code, i don't know. Either way we can be confident this API won't change from under us. We are also no longer importing experimental interfaces exposed from the index of wp components.

Reply:
This was at a time when had other plans for slot fills, including publishing that package and the possibility of it not being pegged.